package.json

```diff
@@ -103,6 +103,8 @@
     "react": "^16.8.0",
     "react-autosuggest": "^9.4.3",
     "react-breadcrumbs-dynamic": "^1.2.0",
+    "react-dnd": "^5.0.0",
+    "react-dnd-html5-backend": "^3.0.2",
```

Comment:
these deps are not explicitly imported anywere in the code - do we need them?

Reply:
Yes, they are required by the tag input control.  If these are missing then the app won't compile.